rcp/org.testeditor.rcp4.views.tcltestrun/plugin.xml

```diff
@@ -6,5 +6,12 @@
             class="org.testeditor.rcp4.views.tcltestrun.ExecutableExtensionFactory:org.testeditor.rcp4.views.tcltestrun.TclLauncherUi">
       </launcher>
    </extension>
-
+	<extension
+	      point="org.eclipse.ui.console.consoleFactories">
+	   <consoleFactory
+	         class="org.testeditor.rcp4.views.tcltestrun.console.TCLConsoleFactory"
```

Comment:
Shouldn't this start with `org.testeditor.rcp4.views.tcltestrun.ExecutableExtensionFactory:`?